apps/snitch_api/config/config.exs

```diff
@@ -41,6 +41,16 @@ config :snitch_api, SnitchApi.Guardian,
 config :snitch_api, frontend_checkout_url: System.get_env("FRONTEND_CHECKOUT_URL")
 config :snitch_api, hosted_payment_url: System.get_env("HOSTED_PAYMENT_URL")
 
+config :sentry,
```

Comment:
Can we have `sentry` config in project root's config? That will prevent duplicate entries in AdminApp and SnitchApi.

Reply:
Moved the Sentry config to project root config.

Reply:
Also we have `sentry` dependencies in all 3 apps. Should we just keep the dependency on `core`, as `admin` and `api` app has dependency on `core`

@pkrawat1 @ramansah Any thought on this?

Reply:
Yes, that makes sense :+1: